docs/sources/administration/configuration.md

```diff
@@ -1222,7 +1226,7 @@ Specifies the type of sampler: `const`, `probabilistic`, `ratelimiting`, or `rem
 
 Refer to https://www.jaegertracing.io/docs/1.16/sampling/#client-sampling-configuration for details on the different tracing types.
 
-Can be set with the environment variable `JAEGER_SAMPLER_TYPE`.
+Can be set with the environment variable `GF_TRACING_JAEGER_SAMPLER_TYPE`.
```

Comment:
I suppose this (and `sampler_param`) does follow the convention already given that it is under `[tracing.jaeger.sampler_type]` but it's called out as `JAEGER_SAMPLER_TYPE` here (when it doesn't need to be, given that it follows the `GF_<section>_<parameter>` convention. So I think the comment could either be updated as here, or removed entirely, but as `JAEGER_SAMPLER_TYPE` I do think it's somewhat confusing.